extensions/amp-mraid/0.1/test/validator-amp4ads-amp-mraid-no-fallback-typo.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
```

Comment:
License and test description here as well.

Reply:
done